util/reggen/uvm_reg_base.sv.tpl

```diff
@@ -6,7 +6,27 @@
   from reggen.access import HwAccess, SwRdAccess, SwWrAccess
   from reggen.multi_register import MultiRegister
   from reggen.register import Register
-%>
+  from typing import Dict
+
+  # Get a list reg and its instance name
+  # For single reg, return (reg, reg.name)
+  # For multireg, if it's dv_compact, return List(mr.reg, mr.name[idx]),
+  # if not, return all the mr.regs with their name
```

Comment:
I don't think this comment is right: I think this function returns a dictionary mapping register instance names (if that's the right phrase?) to their corresponding registers.

Reply:
I used List earlier, forgot to update comments. Fixed, thanks